style.css

```diff
@@ -149,3 +149,310 @@ html,
 .list-social {
   list-style: none;
 }
+
+/* --------STARTS SECTION WORKS MOBILE----------------- */
+
+.works {
+  width: 100%;
+  background-color: white;
+}
+
+.header-works {
+  font-family: 'Crete Round', serif;
+  font-size: 40px;
+  margin-left: 5%;
+  font-weight: 300;
+  color: #1c1a19;
+  margin-top: 20vh;
+}
+
+.division {
+  width: 90%;
+  height: 2px;
+  margin-bottom: 10vh;
+  color: #212121;
+}
+
+.image1 {
+  width: 90%;
+  margin-left: 5%;
+}
+
+.header-posts {
+  margin-left: 5%;
+  margin-top: 3vh;
+  margin-bottom: 2vh;
+  font-family: 'Crete Round', serif;
+  font-weight: lighter;
+  font-size: 32px;
+}
+
+.posts-paragraph {
+  font-family: 'Inter', sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 5%;
+  margin-right: 7%;
+  color: #091e42;
+}
+
+.list-tools {
+  display: flex;
+  padding-left: 0;
+  margin-left: 5%;
+  margin-right: 10%;
+  justify-content: center;
+  align-content: center;
+  gap: 2vh;
+  margin-top: 2vh;
+}
+
+.container-cards {
+  width: 100vw;
+  display: flex;
```

Comment:
Good work , you cards are well aligned but I would ask you to use grid layout here as it was requested in this project requirement : "in order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card"